doc/tutorials/using_windows_as_uos.rst

```diff
@@ -18,9 +18,9 @@ ACRN Service OS Setup
 *********************
 You may refer to the steps in :ref:`getting-started-apl-nuc` for
 Intel NUC to set up ACRN on the KBL NUC. After following the steps in that guide,
-you should be able to launch a Clear Linux UOS successfully.
+you should be able to launch a Clear Linux Guest VM successfully.
```

Comment:
Do you need to install LaaG first, and then WaaG?  Should this sentence instead say "you should be able to launch Windows as a Guest VM successfully"

Reply:
I think "Launch the Service VM" as the precondition is enough for this introduction, so I just change this sentence to:
```
you should be able to launch the Service VM successfully.

```